requirements.txt

```diff
@@ -44,3 +45,5 @@ urllib3==1.22
 uWSGI==2.0.15
 Werkzeug==0.12.2
 wrapt==1.10.11
+
+# git+https://github.com/elifesciences/api-raml.git#egg=elife_api
```

Comment:
Needs removing.

Reply:
Have now replaced with `api-validator-python` url.

`-e git+https://github.com/elifesciences/api-validator-python.git#egg=api-validator-python`

Note the use of the `-e` flag for editable, without this it does not unpack the non Python files into the site-packages directory as by default libraries are set to non editable.